src/OrchardCore.Modules/OrchardCore.Lucene/LuceneIndexManager.cs

```diff
@@ -52,19 +54,19 @@ LuceneAnalyzerManager luceneAnalyzerManager
                 shellOptions.Value.ShellsApplicationDataPath,
                 shellOptions.Value.ShellsContainerName,
                 shellSettings.Name, "Lucene");
-
             _rootDirectory = Directory.CreateDirectory(_rootPath);
+            _luceneAnalyzerSettingsService = luceneAnalyzerSettingsService;
             _luceneAnalyzerManager = luceneAnalyzerManager;
         }
 
-        public void CreateIndex(string indexName)
+        public async Task CreateIndex(string indexName)
         {
-            Write(indexName, _ => { }, true);
+           await Write(indexName, _ => { }, true);
```

Comment:
When a method doesn't run anything after an async call then you don't need to await it, just return the task. In this case the method will become:

```
public Task CreateIndex(string indexName)
{
  return Write(indexName, _ => { }, true);
}
```